package.json

```diff
@@ -10,9 +10,9 @@
     "format": "yarn prettier --write",
     "lint": "eslint ./",
     "validate": "yarn prettier --list-different && yarn tsc && yarn lint",
-    "run-android-bte": "react-native run-android --variant=bteDebug --appId=org.pathcheck.covidsafepathsBte",
+    "run-android-bt": "react-native run-android --variant=btDebug --appId=org.pathcheck.covidsafepathsBt",
```

Comment:
I believe our app id on the play store is `org.pathcheck.covidsafepathsBte`, which means that changing to `org.pathcheck.covidsafepathsBt` will create a new store listing when it is uploaded. Since we haven't launched yet this is ok, just flagging it here

Reply:
Will change it back - thanks!